.eslintrc.json

```diff
@@ -16,5 +16,8 @@
         "@typescript-eslint"
     ],
     "rules": {
+        "camelcase": "off",
+        "@typescript-eslint/no-unused-vars": "error",
+        "@typescript-eslint/naming-convention": "off"
```

Comment:
We can be much more granular here going forward.

This is a temporary solution to avoid errors on Contract factory types that use a weird casing. 

e.g. `LivepeerMock__factory` , combination of PascalCase and snake_case with double undies